package.json

```diff
@@ -16,6 +16,7 @@
     "protocol-buffers-encodings": "^1.2.0",
     "ready-resource": "^1.0.0",
     "safety-catch": "^1.0.2",
+    "same-data": "^1.0.0",
```

Comment:
In which case would `same-data` not be enough? There is also `same-object` (what `brittle` uses)

Just asking because people might use the option to later realize that there are duplicated inserts due `same-data` not supporting the case?